cypress/features/regression/button.feature

```diff
@@ -0,0 +1,83 @@
+Feature: Button component
+  I want to check Button component properties
+
+  Background: Open Button component page default
+    Given I open "Button" component page
+
+  @positive
+  Scenario Outline: Set Button size to <size>
+    When I select size to "<size>"
+    Then Button height is "<height>" and width is "<width>"
+    Examples:
+      | size   | height | width       |
+      | small  | 32px   | 136.4375px  |
+      | medium | 40px   | 152.4375px  |
+      | large  | 48px   | 182.78125px |
+
+  @positive
+  Scenario Outline: Set Button subtext to <subtext>
+    When I select size to "large"
```

Comment:
we should use `Given` instead `When` in this step

Reply:
fix